tests/test_grid3d/test_grid_xtgformats_io.py

```diff
@@ -71,6 +71,32 @@ def read():
     assert_allclose(benchmark_grid._actnumsv, grid2._actnumsv)
 
 
+@pytest.mark.benchmark(group="import/export")
+def test_benchmark_grid_grdecl_export(benchmark, tmp_path, benchmark_grid):
+    fname = join(tmp_path, "reek_geo_grid.grdecl")
+
+    @benchmark
+    def write():
+        benchmark_grid.to_file(fname, fformat="grdecl")
+
+
+@pytest.mark.benchmark(group="import/export")
+def test_benchmark_grid_grdecl_import(benchmark, tmp_path, benchmark_grid):
+    fname = join(tmp_path, "reek_geo_grid.grdecl")
+
+    benchmark_grid.to_file(fname, fformat="grdecl")
+
+    grid2 = xtgeo.Grid()
+
+    @benchmark
```

Comment:
I see this isnt the recommended way of using benchmark, perhaps change it?

Reply:
Added a commit that changes all uses of benchmark to the recommended pattern :)